functions/pair_color_tests/tests_functions.py

```diff
@@ -0,0 +1,28 @@
+from functions.config.color_codes import MAJOR_COLORS, MINOR_COLORS
```

Comment:
File name indicates that the code contained in this file is related to tests. Whereas, these seem to be production code which converts pair number to color code and vice versa.
The file name could be renamed to reflect its intent.

Reply:
I go to made this change in the last commit for today! thanks for the advice